js/scripts.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function() {
```

Comment:
you don't work on DOM elements - so you shouldn't check is document ready

or if you want extra task - check that DOM  is loaded in vanilla JS